### PR TITLE
Use standard charsets

### PR DESCRIPTION
**Pull request by wreulicke**
Use standard charsets

### CODE REVIEW

File: spotbugs/src/main/java/edu/umd/cs/findbugs/charsets/UTF8.java [hunk 1]
```diff
@@ -47,11 +48,6 @@
      *
      */
     private static final String UTF_8 = "UTF-8";
```
Comment (ThrawnCA) [inline spotbugs/src/main/java/edu/umd/cs/findbugs/charsets/UTF8.java]: Could also drop this and use `StandardCharsets.UTF_8.name`

Comment (wreulicke) [reply spotbugs/src/main/java/edu/umd/cs/findbugs/charsets/UTF8.java]: Oh, Thanks!

Comment (wreulicke) [reply spotbugs/src/main/java/edu/umd/cs/findbugs/charsets/UTF8.java]: Fixed! 😄

**Comment by wreulicke** [comment]
@ThrawnCA 
Sorry, I pushed because the CI is failed.
Could you review again?
Thanks.

**Comment by wreulicke** [comment]
@ThrawnCA Thanks for your review! :smile:

**Pull request merged**